test/SILGen/typelowering_inverses.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %target-swift-frontend -emit-silgen -enable-experimental-feature NoncopyableGenerics -disable-availability-checking -module-name main %s | %FileCheck %s
```

Comment:
Because it uses an experimental feature, the test needs a `REQUIRES: asserts` or it will fail in non-assert builds.